pasta_eln/printer.py

```diff
@@ -15,24 +15,23 @@ def createQRcodeSheet(fileName:str="../qrCodes.pdf") -> None:
   import qrcode
   import numpy as np
   from PIL import Image
-  from commonTools import commonTools as cT  # don't import globally since it disturbs translation
-  img = qrcode.make(cT.uuidv4(),
-                    error_correction=qrcode.constants.ERROR_CORRECT_M)
-  size = img.size[0]
+  #from commonTools import commonTools as cT  # don't import globally since it disturbs translation
```

Comment:
remove the comments if not needed!

Reply:
I commented the entire code, such that it can be used in the future when the issue is fixed.